listener/src/event_handler/sellorder_handler.rs

```diff
@@ -0,0 +1,42 @@
+use failure::Error;
+use slog::Logger;
+use std::sync::Arc;
+
+use dfusion_core::models::PendingFlux;
+
+use graph::components::ethereum::EthereumBlock;
+use graph::components::store::EntityOperation;
+use graph::data::store::{Entity};
+
+use web3::types::{Log, Transaction};
+
+use super::EventHandler;
+use super::util;
+
+#[derive(Debug, Clone)]
+pub struct SellOrderHandler {}
+
+impl EventHandler for SellOrderHandler {
+    fn process_event(
+        &self,
+        logger: Logger,
+        _block: Arc<EthereumBlock>,
+        _transaction: Arc<Transaction>,
+        log: Arc<Log>
+    ) -> Result<Vec<EntityOperation>, Error> {
+        let entity_id = util::entity_id_from_log(&log);
+        let flux = PendingFlux::from(log);
```

Comment:
You are creating a PendingFlux, where you actually should create an Order?

Reply:
Yes, PR Order extension is handling this